spec.bs

```diff
@@ -2357,3 +2372,89 @@ achieve the outcomes described in the above explanatory content.
   /fenced-frame/permission-geolocation.https.html
   /fenced-frame/permission-notification.https.html
 </wpt>
+
+<h3 id=private-network-access-monkeypatch>Private Network Access</h3>
+
+[[Private-Network-Access]] is a security feature that regulates when and how [=local network
+requests=] can be made. On [[HTML#creating-browsing-contexts|creation]], <{iframe}>s inherit their
+[=Document/policy container=] from their [=navigable container=]'s [=Node/node document=], while
+<{fencedframe}>s do not. While this prevents information from leaking into a <{fencedframe}> from
+its embedder, it also results in a [=Document/policy container=] for {{Document}}s inside of a
+<{fencedframe}> being created that causes [=subresource requests=] to not do private network access
+checks. This section patches the [[Private-Network-Access]] specification to make subresources be
```

Comment:
Also can you clarify "private network access checks" here? What does this mean? There is the normal preflight which conditionally now deals with PNA headers depending on whether the request is a local network request, is that what you mean? Or specifically the https://wicg.github.io/local-network-access/#local-network-access-check? Or both?

Reply:
Resolving because the spec is now more generalized and this comment is outdated.